.github/workflows/ci.yml

```
@@ -111,6 +111,76 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ steps.generate_token.outputs.token }}
 
+  build-standalone:
+    needs: build
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          path: matrix-neoboard
+          # required for changesets
+          fetch-depth: '0'
+          # don't persist the credentials so the changesets action doesn't use the
+          # github actions token but the git token provided via environment variable
+          persist-credentials: false
+
+      - uses: actions/checkout@v4
+        with:
+          repository: nordeck/matrix-neoboard-standalone
+          path: matrix-neoboard-standalone
+          # required for changesets
+          fetch-depth: '0'
+          # don't persist the credentials so the changesets action doesn't use the
+          # github actions token but the git token provided via environment variable
+          persist-credentials: false
+          ssh-key: ${{ secrets.STANDALONE_DEPLOY_KEY }}
+
+      - name: Setup Node.js
```

Looks like you want to use the verb

```suggestion
      - name: Set up Node.js
```